src/Forms/FixerFormType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the JoliTypo Website project.
+ * (c) JoliCode <coucou@jolicode.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Forms;
+
+use App\Services\FixerDataHolder;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class FixerFormType extends AbstractType
```

Comment:
We do not put `Form` for Type, It should be `FixerType`. But I don't like this name.
It's not about about one Fixer, but about configuring and fixing content.